pom.xml

```diff
@@ -18,22 +18,76 @@
             checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
         <java.version>17</java.version>
+        <lombok.mapstruct.binding.version>0.2.0</lombok.mapstruct.binding.version>
+        <mapstruct.version>1.5.5.Final</mapstruct.version>
     </properties>
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Please, make PR with changes that are related to the certain task.
See the task description.
<img width="683" alt="image" src="https://github.com/flekros/book-store/assets/140437083/75c10eca-5943-4714-a073-90e5886933ea">